server/odc-service/src/main/java/com/oceanbase/odc/service/pldebug/session/AbstractDebugSession.java

```diff
@@ -113,19 +117,50 @@ protected void acquireNewConnection(ConnectionSession connectionSession,
 
     protected DebugDataSource acquireDataSource(ConnectionSession connectionSession, List<String> initSqls) {
         ConnectionConfig config = (ConnectionConfig) ConnectionSessionUtil.getConnectionConfig(connectionSession);
-        DebugDataSource dataSource = new DebugDataSource(config, initSqls);
         String schema = ConnectionSessionUtil.getCurrentSchema(connectionSession);
-        String host;
-        Integer port;
-        if (StringUtils.isBlank(config.getClusterName())) {
-            host = config.getHost();
-            port = config.getPort();
-        } else {
+        String host = config.getHost();
+        Integer port = config.getPort();
+        String url;
+        if (connectionSession.getConnectType() == ConnectType.OB_ORACLE) {
+            if (StringUtils.isBlank(config.getClusterName())
+                    && connectionSession.getConnectType() != ConnectType.CLOUD_OB_ORACLE) {
```

Comment:
You could delete this line `&& connectionSession.getConnectType() != ConnectType.CLOUD_OB_ORACLE `

Reply:
ok